Yona/Yona/Notifications/MeDashBoardMainViewController.swift

```diff
@@ -102,7 +102,8 @@ class MeDashBoardMainViewController: YonaTwoButtonsTableViewController {
                 self.avtarImg.kf.setImage(with: URL, placeholder: nil, options: nil, progressBlock: nil, completionHandler: { (image, error, cacheType, imageURL) -> () in
                     let btnName = UIButton()
                     btnName.frame = CGRect(x:0, y:0, width:32, height:32)
```

Comment:
Use a constant for t size of the square, so the number (32) doesn't need to be repeated multiple times.

Reply:
Created constant for height and width in Constants.swift and using it.